packages/pubsub/src/Providers/AWSIot.ts

```diff
@@ -1,41 +1,43 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
-import { MqttOverWSProvider, MqttProviderOptions } from './MqttOverWSProvider';
-import { Signer, Credentials } from '@aws-amplify/core';
-
+import { MqttOverWS, MqttOptions } from './MqttOverWS';
+import { Signer } from '@aws-amplify/core/internals/utils';
```

Comment:
We are going to remove `Signer` class soon. Can you use `preSignUrl` instead? Signer was intended for backwards compabibility over functional `presignUrl`;


```suggestion
import { presignUrl } from '@aws-amplify/core/internals/aws-client-utils';
```

Reply:
I made a run at the requested migration. There aren't examples of this and Signer is doing a lot of enrichment / data cleaning that isn't done by presignUrl.  Can you provide some clarification / examples of how this migration is supposed to go?

Reply:
We talked this over. The change to make this happen is more involved than makes sense to pull into this change.